docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -122,6 +122,43 @@ Elliptic Curve Signature Algorithms
         :returns: A new instance of a :class:`EllipticCurvePublicKey`
             provider.
 
+Elliptic Curve Key Exchange algorithm
+-------------------------------------
+
+.. class:: ECDH(algorithm)
```

Comment:
I have no objection to the interface supporting both. I just see value in having a version of it that automatically creates a random key for you if you don't want to pass in an existing private key. However, it's only a handful of lines of code to build such a wrapper, so I can live with the API that's here.